src/GafferSceneUIModule/LightEditorBinding.cpp

```diff
@@ -173,20 +184,30 @@ class InspectorColumn : public PathColumn
 			ScenePlug::PathScope scope( scenePath->getContext(), &scenePath->names() );
 			scope.setCanceller( canceller );
 
-			Inspector::ConstResultPtr inspectorResult = m_inspector->inspect();
-			if( !inspectorResult )
+			try
 			{
-				return result;
-			}
+				Inspector::ConstResultPtr inspectorResult = m_inspector->inspect();
+				if( !inspectorResult )
+				{
+					return result;
+				}
 
-			result.value = runTimeCast<const IECore::Data>( inspectorResult->value() );
-			/// \todo Should PathModel create a decoration automatically when we
-			/// return a colour for `Role::Value`?
-			result.icon = runTimeCast<const Color3fData>( inspectorResult->value() );
-			result.background = g_sourceTypeColors.at( (int)inspectorResult->sourceType() );
-			if( auto source = inspectorResult->source() )
+				result.value = runTimeCast<const IECore::Data>( inspectorResult->value() );
+				/// \todo Should PathModel create a decoration automatically when we
+				/// return a colour for `Role::Value`?
+				result.icon = runTimeCast<const Color3fData>( inspectorResult->value() );
+				result.background = g_sourceTypeColors.at( (int)inspectorResult->sourceType() );
+				if( auto source = inspectorResult->source() )
+				{
+					result.toolTip = new StringData( "Source : " + source->relativeName( source->ancestor<ScriptNode>() ) );
+				}
+			}
+			catch( const std::exception& e )
```

Comment:
If we take this approach, it seems that every custom column type is going to be responsible for handling exceptions and turning them into the same CellData with an error icon and message. Is there any reason we can't do this once, centrally in [`PathModel::Item::updateData()`](https://github.com/GafferHQ/gaffer/blob/main/src/GafferUIModule/PathListingWidgetBinding.cpp#L1187-L1196)? This wouldn't preclude a column from doing its own exception handling if it could do a better job somehow, but it would mean that most column types wouldn't have to deal with exceptions at all.

Reply:
Fixed in 3c396c6a36538cf4a96a2a4188013d1909b477dd